native/src/components/AddressInfo.tsx

```diff
@@ -0,0 +1,61 @@
+import Clipboard from '@react-native-clipboard/clipboard'
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Platform, TouchableOpacity } from 'react-native'
+import styled from 'styled-components/native'
+
+import { getExternalMapsLink, LocationModel } from 'api-client'
+
+import ExternalLinkIcon from '../assets/ExternalLink.svg'
+import { contentDirection } from '../constants/contentDirection'
+import useSnackbar from '../hooks/useSnackbar'
+import openExternalUrl from '../utils/openExternalUrl'
+import { reportError } from '../utils/sentry'
+import Text from './base/Text'
+
+const Container = styled.View<{ language: string }>`
+  justify-content: space-between;
+  flex-direction: ${props => contentDirection(props.language)};
+`
+
+const IconContainer = styled.TouchableOpacity`
+  align-self: center;
+  padding: 0 8px;
+`
+
+type AddressInfoProps = {
+  location: LocationModel<number>
+  language: string
+}
+
+const AddressInfo = ({ location, language }: AddressInfoProps): ReactElement => {
```

Comment:
Renamed from PoiDetailItem. Not 100% happy with the name so if you have better ideas let me know :)

Reply:
Only other one I can come up with is AddressBox, and I think AddressInfo is better.